bt_sim:
  gui: True
  gripper_force: 10
  scene: random
  topic: /sim_complete
  # scene: as_test_scene.yaml

hw:
  scene_file: $(find active_search)/cfg/hw/scene01.yaml

grasp_controller:
  arm_id: fr3
  base_frame_id: fr3_link0
  ee_grasp_offset: [0.0, 0.0, -0.383, 0.924, 0.0, 0.0, 0.065]  # offset to panda_link8
  # grasp_drop_config: [2.0, 0.0, -0.383, 0.924, 0.0, 0.0, 0.065] 
  grasp_drop_config: [0.0, -0.79, 0.0, -2.356, 0.0, 1.57, 0.79]
  control_rate: 30
  linear_vel: 0.05
  camera:
    frame_id: wrist_camera_depth_optical_frame
    info_topic: /wrist_camera/depth/camera_info
    depth_topic: /wrist_camera/depth/image_rect_raw
    min_z_dist: 0.3

cartesian_velocity_controller:
  topic: /cartesian_velocity_controller/set_command
